src/uct/ib/mlx5/rc/rc_mlx5_common.c

```diff
@@ -594,6 +594,68 @@ void uct_rc_mlx5_release_desc(uct_recv_desc_t *self, void *desc)
     ucs_mpool_put_inline(ib_desc);
 }
 
+ucs_status_t
+uct_rc_mlx5_dp_ordering_ooo_init(uct_rc_mlx5_iface_common_t *iface,
+                                 uct_ib_mlx5_md_t *md, uint64_t tl_flag,
```

Comment:
no need to pass md, can get it from the iface

Reply:
fixed